lib/protoflow-blocks/src/system.rs

```diff
@@ -165,9 +165,28 @@ impl HashBlocks for System {}
 
 #[cfg(feature = "hash")]
```

Comment:
This feature guard won't work properly; I'd think it needs to be a logical OR between the individual "hash-" features?